src/libraries/System.Net.HttpListener/src/System/Net/Managed/HttpListenerTimeoutManager.Managed.cs

```diff
@@ -36,6 +36,7 @@ public TimeSpan IdleConnection
             }
         }
 
+        [UnsupportedOSPlatform("browser")]
```

Comment:
The marked methods in this file also look set specific. Since they are already marked windows-specific, I believe we can just omit these 4 APIs in both the ref file and here.